examples/player/simid_player.html

```diff
@@ -35,11 +35,43 @@
       simidPlayer.initializeAd();
     }
 
+    function disableButtons(state) {
+      let adRequests = document.querySelectorAll(".ad_request");
+      adRequests.forEach((request) => {
+        request.disabled = state;
+      });
+    }
+
     function playAd() {
       if (!simidPlayer) {
         initAd();
       }
       simidPlayer.playAd();
+      disableButtons(false);
```

Comment:
When you use a hard-coded value as a param in a function call, we usually try to add an inline comment saying what that value is representing, so in this case

`disableButtons(/* state= */ false);`

Reply:
Cool! I've never seen this before